hazelcast/src/test/java/com/hazelcast/jet/impl/JetClientInstanceImplTest.java

```diff
@@ -39,8 +39,8 @@
     @Test
     public void given_singleMapOnMember_when_getDistributedObjectsCalled_then_ReturnedObjectInfo() {
         // Given
-        JetInstance member = createJetMember();
-        JetClientInstanceImpl client = (JetClientInstanceImpl) createJetClient();
+        HazelcastInstance member = createHazelcastInstance();
+        JetClientInstanceImpl client = (JetClientInstanceImpl) createHazelcastClient().getJet();
```

Comment:
Do we still need this class(`JetClientInstanceImpl`) anymore? Or can we deprecate this class?

Reply:
Those classes now work as an implementation of the `JetService` interface, and when `Hazelcast#getJet()` is called on the client side, this `JetClientInstanceImpl` is obtained. I also think the naming here is a bit confusing, but at first I chose not to rename it to show that it shouldn't leave this `JetInstance` hierarchy yet.
Although we deprecate `JetInstance`, we still have `JetInstance` in this hierarchy of Jet classes to not break its old usages. See https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/jet/impl/AbstractJetInstance.java#L60-L67.

Reply:
`JetClientInstanceImpl ` -> `JetServiceImpl` WDYT?

Reply:
I also think that implementation classes can be renamed, but I think it would be better to discuss this in a separate PR. But, I don't think `JetService` -> `Jet` change can be performed since Jet is used before.